packages/ses/NEWS.md

```diff
@@ -20,7 +20,14 @@ User-visible changes in SES:
   world, and also allows its `t.throws` assertion to work. `tape` (version
   5.x) still has problems. [#293]
 
-  [#293]: https://github.com/Agoric/SES-shim/issues/293)
+  [#293]: https://github.com/Agoric/SES-shim/issues/293
+* Replaces the old `noTame*` options to `lockdown` with new `*Taming` options.
```

Comment:
nit: Maybe add a new line between 23 and 24, or possibly just consolidate the link refs at the bottom of the block.